http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -57,7 +62,9 @@ class MonitorAtBoot[F[_]](publisherQueue: Queue[F, LeoPubsubMessage],
       .parEvalMapUnordered(10) { case (a, n, c) =>
         for {
           now <- F.realTimeInstant
-          implicit0(traceId: Ask[F, TraceId]) = Ask.const[F, TraceId](TraceId(s"BootMonitor${now}"))
+          implicit0(traceId: Ask[F, AppContext]) = Ask.const[F, AppContext](
```

Comment:
any reason why we need `AppContext` here?

Reply:
Yeah to string it through to handleApp, there is a downstream fn that requires an implicit AppContext